### DISCUSSION TITLE
How does one get the number of elements/size of a ControlledList?

### DISCUSSION THREAD

**Discussion by iron3oxide** [lowdefy/lowdefy#1233] [Questions and Help] [answered]
Sorry for starting so many discussions, I am not yet well acquainted with lowdefy and can't find an answer in the docs or in other discussions. On the plus side, my questions will hopefully help the next person find their answers faster, so one way to view it would be "building the knowledgebase".

As previously mentioned, I use a ControlledList to give the user the option to add filters to their query. For design reasons, this list is hidden in a Collapse. Since a user might rehide the filter list and thus forget they have filters set, I wrapped the Collapse in a badge which should display the number of filters set (or nothing if the list is empty). I know that ControlledLists are contolled by state, but not how I get the number of elements of the list as a 'string', so that it can be the value of badge.count

If this is not (easily) achievable, I would also settle for a solution where the badge merely displays whether the list is empty or not. Thanks in advance!

**Comment by Gervwyk** [comment, marked as answer]
Hi @iron3oxide 

You can just do:

```yaml
...
properties:
  count:
    _state: my_list.length
```
